master/getting-started/docker/installation/manual.md

```diff
@@ -18,46 +18,47 @@ title: Installing Calico for Docker
    sudo chmod +x /usr/local/bin/calicoctl
    ```
 
-2. Configure access to your etcd cluster, [calicoctl - etcd datastore](/{{page.version}}/reference/calicoctl/setup/etcdv3).
-3. Launch `calico/node`:
+1. Configure access to your etcd cluster, [calicoctl - etcd datastore](/{{page.version}}/reference/calicoctl/setup/etcdv3).
```

Comment:
Looks like `{{site.baseurl}}` is missing here?

Reply:
This is purposeful, a relative link, which is more portable